### PR TITLE
Requesting to merge David T text changes into Dev branch 

### PR DESCRIPTION
**Pull request by DavidThompson2**
## Proposed changes

Looking to merge Dillon's changes to Dev. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

@CTUCKER0900 

Tag your reviewer here to ensure they get a notification!

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -2,4 +2,9 @@ This is a random text file. You can add and remove things to it to test how git
 
 
 This is some text.
-I am just normally editing a text file here.
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:5]: Did you mean to delete this?

File: text_file.txt [hunk 2]
```diff
@@ -2,4 +2,9 @@ This is a random text file. You can add and remove things to it to test how git
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:2]: Remove this empty space

**Review by DIPAD-Fran-Bryden — COMMENTED** [review]
Good code!

File: text_file.txt (hunk 1 above)
Comment (DillonDfT) [inline text_file.txt:5]: Why has this been deleted?

**Review by DillonDfT — CHANGES_REQUESTED** [review]

File: text_file.txt (hunk 1 above)
Comment (DavidThompson2) [reply text_file.txt:5]: Because it is not relevant to KitKat time

File: text_file.txt [hunk 3]
```diff
@@ -2,4 +2,9 @@ This is a random text file. You can add and remove things to it to test how git
 
 
 This is some text.
```
Comment (CTUCKER0900) [inline text_file.txt:4]: Did you mean to keep this?

File: text_file.txt [hunk 4]
```diff
@@ -2,4 +2,9 @@ This is a random text file. You can add and remove things to it to test how git
...
-I am just normally editing a text file here.
+
+
+
+KitKat time
```
Comment (CTUCKER0900) [inline text_file.txt:8]: KitKats are great!

**Review by CTUCKER0900 — COMMENTED** [review]
Fantastic - love KitKats

**Pull request closed without merging**